parts/ch11.tex

```diff
@@ -77,7 +79,7 @@ \subsection{Соответствие между мозгами}
 \subsection{Загадка местоположения мозговых процессов}
 
 В попытке найти ответ на этот вопрос, невролог Карл Лашли провел длинную серию экспериментов. В этих экспериментах, начавшихся около 1920 года и продолжавшихся много лет, он попытался обнаружить, где в мозгу у крысы хранится её опыт по прохождению лабиринтов. В своей книге «Мыслящий мозг» Стивен Роуз описывает злоключения Лашли:
-
+ %TODO: как-то оформить цитату
```

Comment:
Leave a blank line above the block! Two consecutive \n end a paragraph (\par), but here you suppress \par with a comment.

Reply:
Will fix it.